examples/tagging/variables.tf

```diff
@@ -0,0 +1,55 @@
+#####################################################
+# IBM Cloud Resource Management
+# Copyright 2021 IBM
+#####################################################
+
+##################################################
+# Resource Group Variables
+##################################################
+variable "ibmcloud_api_key" {
+  description = "IBM Cloud API Key"
+}
+
+variable "is_new_resource_group" {
```

Comment:
Why we need variables related to resource_group here

Reply:
Used variables in main.tf